Assets/HoloToolkit/Input/Scripts/GestureManipulator.cs

```diff
@@ -47,7 +47,7 @@ private void OnEnable()
 
         private void OnDisable()
         {
-            if (GestureManager.Instance)
+            if (GestureManager.Instance != null)
```

Comment:
We might want to get @killerantz to update that in his current PR https://github.com/Microsoft/HoloToolkit-Unity/pull/241 as well?